include/grpcpp/impl/codegen/server_callback.h

```diff
@@ -348,7 +348,8 @@ class ServerBidiReactor : public internal::ServerReactor {
 
  private:
   friend class ServerCallbackReaderWriter<Request, Response>;
-  void BindStream(ServerCallbackReaderWriter<Request, Response>* stream) {
+  virtual void BindStream(
```

Comment:
non-abstract virtual functions in the private section probably deserve a comment, IMO. Because although ServerBidiReactor (and the other classes changed here) are meant to be overridden by the application-level programmer, private functions are not meant as such.